src/plot/plotStore.ts

```diff
@@ -29,6 +31,8 @@ export interface PlotConfig {
 export const usePlotStore = create<PlotConfig>((set) => ({
   detector: true,
   detectorColor: { r: 144, g: 19, b: 254, a: 0.4 },
```

Comment:
could: extract rgba color into a typed value and name the value by the color name